### DISCUSSION TITLE
Please allow for "internal" visibility on organization profile

### DISCUSSION THREAD

**Discussion by hbomark** [community/community#36928] [Repositories]
Per [GH's docs](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-member-only-organization-profile-readme).

One may add a private pinned note by adding a README to
`<organization-name>/.github-private/blob/main/profile/README.md`, where `.github-private` is a repository.

Or, one may add a public note.

However, one is not able to have ["internal" visibility](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/about-repositories#about-internal-repositories) for a profile. May we please have "internal" level visibility for organization profiles? It causes a ton of confusion for users that are in the same enterprise since it tricks them into believing they a member of the org because they can see internal repos, but they actually aren't.

Visually speaking, from the org page, where you can view as either a member or public, that would entail adding "internal" as a possible choice in this dropdown:
![screenshot of the viewing repo page which shows dropdown as either public or member but no internal view](https://user-images.githubusercontent.com/47125187/197309456-17d673cf-98da-4cdd-b47f-1aa4d7ecf368.png)

=== searches done === 
1- https://github.com/orgs/community/discussions?discussions_q=visibility+organization+profile
2- https://github.com/orgs/community/discussions?discussions_q=visibility+organization+profile+internal

I do not see a similar request in the first few results.

Thank you!

**Comment by ossanna16** [comment]
Hi there @hbomark and welcome to our community 😄. Thanks for the suggestion!

**Comment by mbainter** [comment]
Not just view-as, but also the ability to have a different profile for Internal.
